### ISSUE TITLE
[A11Y] Slides carousel: Slides not navigable by screen readers without `controls` attribute present

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5487]
without `controls` attribute screen reader users lose access to the next/previous buttons (because their visibility is set to hidden) and therefore have no way of navigating the carousel.

We need to hide next/previous buttons differently (can't do `display:none` or `visibility:hidden`). One approach is moving them off-screen, another approach is using `opacity:0` (my recommendation)

Applies to both `type="carousel"` and `type="slides"`

**Comment by camelburrito** [comment]
we should give them a -ve z-index and set them to display: none only when there are no next/prev actions available.

**Comment by aghassemi** [comment]
I am adding a a core `-amp-screen-reader` class that can be used to visibly hide elements but keep them accessible. We should use the same class here ( and other similar places) PR coming soon.

**Comment by aghassemi** [comment]
/to @camelburrito 
Please wait for https://github.com/ampproject/amphtml/pull/5542 and reuse the `-amp-screen-reader` to fix this.

**Comment by aghassemi** [comment]
@camelburrito Do you have any free cycles for this during FixIt? (if not, I can reassign)

**Comment by aghassemi** [comment]
Related:  https://github.com/ampproject/amphtml/pull/6333

**Comment by adelinamart** [comment]
This is a P1 that was not updated in a while. Can you provide a status here. Thank you.

**Comment by aghassemi** [comment]
PR is the works: https://github.com/ampproject/amphtml/pull/8822 lack of proper support in mobile screen readers is adding complications.

**Comment by aghassemi** [comment]
de-duping in favour of https://github.com/ampproject/amphtml/issues/7975
